packages/@adobe/spectrum-css-temp/components/alertbanner/index.css

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+.spectrum-AlertBanner {
+  box-sizing: border-box;
+  display: inline-flex;
+  flex-direction: row;
+  align-items: stretch;
+  width: 100%;
+
+  padding-block-start: var(--spectrum-alertbanner-padding-y, var(--spectrum-global-dimension-size-100));
+  padding-inline-end: var(--spectrum-alertbanner-padding-right, var(--spectrum-global-dimension-size-100));
+  padding-block-end: var(--spectrum-alertbanner-padding-y, var(--spectrum-global-dimension-size-100));
+  padding-inline-start: var(--spectrum-alertbanner-padding-left, var(--spectrum-global-dimension-size-200));
+
+  font-size: var(--spectrum-alertbanner-text-size);
+  -webkit-font-smoothing: antialiased;
+}
+
+.spectrum-AlertBanner-typeIcon {
+  flex-shrink: 0;
+  flex-grow: 0;
+
+  margin-block-start: var(--spectrum-alertbanner-icon-padding-y, var(--spectrum-global-dimension-size-85));
+  margin-inline-end: var(--spectrum-alertbanner-icon-padding-right, var(--spectrum-global-dimension-size-150));
+  margin-block-end: var(--spectrum-alertbanner-icon-padding-y, var(--spectrum-global-dimension-size-85));
+  margin-inline-start: 0;
+}
+
+.spectrum-AlertBanner-content {
+  flex: 1 1 auto;
+  box-sizing: border-box;
+  padding-block-start: var(--spectrum-global-dimension-size-65);
+  padding-block-end: var(--spectrum-global-dimension-size-65);
+  padding-inline-start: 0;
+  text-align: start;
+}
+
+.spectrum-AlertBanner-buttons {
+  display: flex;
+  flex: 0 0 auto;
+  align-items: flex-start;
+
+  .spectrum-Button,
+  .spectrum-ClearButton {
+    & + .spectrum-Button,
+    & + .spectrum-ClearButton {
+      margin-inline-start: var(--spectrum-alertbanner-button-gap);
+    }
+  }
+}
+
+.spectrum-AlertBanner-body {
+  flex: 1 1 auto;
+  align-self: center;
+
+  /* https://spectrum.adobe.com/page/alert-banner/#Text-overflow */
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
text overflow is different, i don't think we want wrap here
this should all go away though with my other comment about moving the action button up so everything is a sibiling